mycroft/client/speech/mic.py

```diff
@@ -199,7 +213,7 @@ def __init__(self, wake_word_recognizer):
 
     @staticmethod
     def record_sound_chunk(source):
-        return source.stream.read(source.CHUNK)
+        return source.stream.read(source.CHUNK, self.overflow_exc)
```

Comment:
self. flops in the staticMethod.  Need to add this as a class member that defaults to None, then switch here and the constructor to use  ```ResponsiveRecognizer.overflow_exc``` instead of ```self.overflow_exc```

Reply:
Thwarted again by my nemesis `staticmethod`, will fix.

Reply:
In the end I found the cleanest way to fix this to be just removing the `@staticmethod` decorator. There was no particular reason for this to not be a proper class method as far as I could see.